### ISSUE TITLE
I got confused about the transform code  of computing FID

### ISSUE THREAD

**Issue by Samuel-Recovery-123** [ankanbhunia/PIDM#70]
https://github.com/YanzuoLu/CFLD/blob/9892b2fd88ec05327b6a825182388f7185daf454/models/metrics.py#L89
```
    def forward_inception(self, x):
        x = F.interpolate(x, size=(299, 299), mode='bilinear')
        x[:, 0] = x[:, 0] * (0.229 / 0.5) + (0.485 - 0.5) / 0.5
        x[:, 1] = x[:, 1] * (0.224 / 0.5) + (0.456 - 0.5) / 0.5
        x[:, 2] = x[:, 2] * (0.225 / 0.5) + (0.406 - 0.5) / 0.5
        out = self.inception_blocks(x)
        return out
```
According to the context, x is normalized to [0, 1]. How does this transform means? It seems like denormalization from standard normalization on ImageNet with std [0.229 , 0.224,  0.225] and mean [0.485, 0.456, 0.406] and normalization to [-1, 1] subsequently. I referred to the official implementation code of FID computing and found that the image is normalized to [0, 1] preliminarily, https://github.com/mseitzer/pytorch-fid/blob/b9c18118d082cbd263c1b8963fc4221dc1cbb659/src/pytorch_fid/fid_score.py#L146. Could you please explain it?

**Comment by ankanbhunia** [comment]
Inception expects images to be in range [-1,1]. 

I think the official impl also do normalization inside the forward pass.

see [here](https://github.com/mseitzer/pytorch-fid/blob/b9c18118d082cbd263c1b8963fc4221dc1cbb659/src/pytorch_fid/inception.py#L151).

**Comment by Samuel-Recovery-123** [comment]
Thanks for your  reply! I should have commented on the work of CFLD, which is quite similar with yours. It's my mistake. Really sorry!
